models/articles.models.js

```diff
@@ -31,4 +31,20 @@ const findArticles = () => {
       return rows;
     });
 };
-module.exports = { findArticle, findArticles };
+
+const findCommentsById = (id) => {
+  return db
+    .query(
+      `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`,
+      [id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0)
+        return Promise.reject({
+          status: 404,
+          message: "This id does not exist",
+        });
```

Comment:
This isn't accurate as the article might exist but there might no be comments associated which would be a valid request and we want to return the empty array.